Runtime/EffectSystem/ScriptableObjects/GameplayEffectSO.cs

```diff
@@ -30,7 +30,7 @@ public class GameplayEffectSO : ScriptableObject, IGameplayEffectDef
         [field: SerializeField, Tooltip("What attribute to affect and how it affected")]
         public EffectDetails EffectDetails { get; private set; } = new();
 
-        [field: SerializeField]
+        [field: SerializeReference, SubclassSelector, Tooltip("Stacking effect when this effect is applied on the target")]
```

Comment:
SubclassSelector
It's a struct so this might not needed

Reply:
I get it. Done!